### ISSUE TITLE
Crash when selecting any node

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1147]
**[Submitted to the original trac issue database at 5.22am, Friday, 5th September 2008]**

Using latest released windows version (0.11) with an executable build from latest SVN. Whenever a node is selected the program crashes.

System is Vista x64 and executable built using latest QT (4.4.1) with included MinGW

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 11.54am, Friday, 5th September 2008]**

Well, basically, no.
Merkaartor doesn't "Crash when selecting any node".

First, what do you mean by "Using latest released windows version (0.11) with an executable build from latest SVN"? Is it the released version or a self-build one?

Second, try to delete the registry key "HKCU\Software\BartVanhauwaert", to be sure.

Try again. If it crashes 100%, it's a problem with your build.
Otherwise, there is a context I need to know.

**Comment by tomhughes** [comment]
**Author: tommyc_81@hotmail.com**
**[Added to the original trac issue at 12.52pm, Friday, 5th September 2008]**

Version installed was the 0.11 release for windows using the supplied installer. On top of that I've installed QT and MinGW and made builds from latest SVN and replaced the merkaartor.exe supplied with the 0.11 release with the one from my build of the executable. (Since the 0.11 release had the extremly disturbing problem that it didn't update roads when nodes were added/removed from it). This has worked fine until about one week ago when the problem started.

Deleting the registry key: "HKCU\Software\BartVanhauwaert?" didn't solve the problem. But removing all supplied .dll's from the Merkaartor folder made it work again (having PATH set to the appropiate QT and MinGW \bin folders).
